modules/ROOT/pages/api-groups-modifying-groups.adoc

```diff
@@ -0,0 +1,97 @@
+= Modify API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your business requirement changes, you might want to modify an API Group by adding new API instances to it. To cover new scenarios, you might want to add a new API instance. 
+
+You can also version an existing instance to add customizations. If a service is no longer required, you can remove the respective API instance that supports the service.
+
+The following tasks are covered in this section:
+
+* <<adding-api-instances-to-groups, Add API instances to Existing API Groups>>
+* <<removing-api-instances-from-groups, Remove API instances from an API Group>>
+* <<creating-new-versions-of-groups, Create new versions of an API Group>>
+
+[[adding-api-instances-to-groups]]
+== Add Instances to Existing API Groups
```

Comment:
We should mention that there are several rules for doing this.

User can always add new API instances (from existing APIs in the group) except the group instance has contracts. If it has contracts, there is nothing he can do.
If one of these scenarios happens, the user can add API instances but NOT APIs:
- The group instance is published
- The group version is published
- There is more than one group instance from the same version

If none of those scenarios happen, then user is free to modify the group instance.

Reply:
Done! Thanks!